stories/Skylign.stories.js

```diff
@@ -0,0 +1,80 @@
+import React, { useEffect } from 'react';
+
+import Skylign from 'skylign';
+import loadWebComponent from 'utils/load-web-component';
+
+export default {
+  title: 'InterPro UI/Skylign',
+};
+const data = {
+  alphabet: 'aa',
+  max_height_theory: 6.453114921518104,
+  max_height_obs: 5.2996166606681,
+  min_height_obs: 0,
+  height_arr: [
```

Comment:
Maybe generate/replicate `height_arr` and `probs_array` so it has enough length that scrolling over the component is necessary